plugins/auto-sizes/includes/improve-calculate-sizes.php

```diff
@@ -82,10 +95,11 @@ function auto_sizes_filter_image_tag( $content, array $parsed_block, WP_Block $b
 		 */
 		$filter = static function ( $sizes, $size ) use ( $block ) {
 
-			$id            = isset( $block->attributes['id'] ) ? (int) $block->attributes['id'] : 0;
-			$alignment     = $block->attributes['align'] ?? '';
-			$width         = isset( $block->attributes['width'] ) ? (int) $block->attributes['width'] : 0;
-			$max_alignment = $block->context['max_alignment'] ?? '';
+			$id                       = isset( $block->attributes['id'] ) ? (int) $block->attributes['id'] : 0;
+			$alignment                = $block->attributes['align'] ?? '';
+			$width                    = isset( $block->attributes['width'] ) ? (int) $block->attributes['width'] : 0;
+			$max_alignment            = $block->context['max_alignment'] ?? '';
+			$container_relative_width = $block->context['container_relative_width'] ?? '1.0';
```

Comment:
This is not used here, so a bit strange to include, also why is it a string?

That said, I see you address all this in #1818, so it's probably okay.

Reply:
Removed in https://github.com/WordPress/performance/pull/1795/commits/085d98e8ffeff15691fbe8d45cce29d91dfaa715